src/dataexplorer/dataexplorer-application/src/components/PatientQuestionnaireResponses/PatientQuestionnaireResponses.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import useFetch from 'utils/hooks/useFetch';
-import { IQuestionnaireResponses } from 'types/IQuestionnaireResponse';
+import { IQuestionnaireResponse } from 'types/IQuestionnaireResponse';
 
 const PatientQuestionnnaireResponses = ({ patientID }: any) => {
-    const { response: questionnaireResponses } = useFetch<
-        IQuestionnaireResponses
+    const { response: patientQuestionnaireResponses } = useFetch<
+        IQuestionnaireResponse
     >('fhir/QuestionnaireResponse?subject=Patient/' + patientID);
-    if (questionnaireResponses) {
+    if (patientQuestionnaireResponses) {
         return (
             <h1>
-                Pasienten har {questionnaireResponses.total} utfylte skjemaer
+                Pasienten har {patientQuestionnaireResponses.entry.length}{' '}
```

Comment:
Remove formatting fragment "{'  '}" ?

Reply:
It comes back every time I try to remove it. But can make an extra line if we want to avoid it